spa/src/components/MainLayout.test.tsx

```diff
@@ -0,0 +1,155 @@
+import { ComponentType } from 'react';
+import { render, screen, within } from 'test-utils';
+import isContributorAppPath from 'utilities/isContributorAppPath';
+import isPortalAppPath from 'utilities/isPortalAppPath';
+import useSubdomain from 'hooks/useSubdomain';
+import MainLayout from './MainLayout';
+
+jest.mock('./analytics/AnalyticsContext', () => ({
+  AnalyticsContextProvider: ({ children }: { children: ComponentType }) => (
+    <div data-testid="mock-analytics-context-provider">{children}</div>
+  )
+}));
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  Suspense: ({ children }: { children: ComponentType }) => <div data-testid="mock-suspense">{children}</div>
+}));
+jest.mock('elements/modal/GlobalConfirmationModal', () => ({
+  __esModule: true,
+  default: ({ children }: { children: ComponentType }) => (
+    <div data-testid="mock-global-confirmation-modal">{children}</div>
+  )
+}));
+jest.mock('utilities/isContributorAppPath');
+jest.mock('utilities/isPortalAppPath');
+jest.mock('hooks/useSubdomain');
+jest.mock('components/authentication/ReauthModal', () => () => <div data-testid="mock-reauth-modal" />);
+jest.mock('components/DashboardRouter', () => ({
+  __esModule: true,
+  default: () => <div data-testid="mock-dashboard-router" />
+}));
```

Comment:
```suggestion
jest.mock('components/DashboardRouter', () => () => <div data-testid="mock-dashboard-router" />));
```
And the same thing for other usage of `__esModule` here.